csrc/scheduler/utils.h

```diff
@@ -848,6 +848,8 @@ TensorView* getUpCastInputOf(const TensorView* buffer_tv);
 //! See device_lower/analysis/tensor_producer_aliases.h
 TensorView* scheduleInputToSkipIntermediates(TensorView* tv);
 
+// Returns true if any of the domains of the tensor is symbolic
+bool isConcreteTensor(const TensorView* tv);
```

Comment:
nitpick, maybe we can just call this `SymbolicTensor`? since the code comment is saying that already.